ansible/roles/stack-sunbird/templates/content-service_application.conf

```diff
@@ -375,7 +375,7 @@ content {
   }
   h5p {
     library {
-      path: "https://s3.ap-south-1.amazonaws.com/ekstep-public-prod/content/templates/h5p-library-latest.zip"
+      path: "https://sunbirddev.blob.core.windows.net/sunbird-content-dev/h5p-standalone-1.3.4.zip"
```

Comment:
why are we hardcoding this value? please make a variable and define it in [defaults/main.yaml](https://github.com/project-sunbird/sunbird-devops/blob/release-3.4.0/kubernetes/ansible/roles/sunbird-deploy/defaults/main.yml) of

Reply:
@rjshrjndrn - it is static for all the environments.